.gitlab-ci.yml

```diff
@@ -35,60 +33,35 @@ unit_tests:
   script:
   - make test
 
-lint_code:
-  stage: test
-  tags: [ "runner:main", "size:large" ]
-  script:
-  - make validate
-
 generate_code:
   stage: test
   tags: [ "runner:main", "size:large" ]
   script:
-  - make generate
+  - make generate manifests
   - git diff --exit-code
 
 build_image:
   stage: image
   tags: [ "runner:docker", "size:large" ]
   image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/docker-notary:0.6.1
+  before_script: []
   script:
-    - IMAGE_NAME=$DOCKER_REGISTRY/$PROJECTNAME:$TARGET_TAG
-    - docker build --build-arg TAG=$TARGET_TAG -t $IMAGE_NAME -f Dockerfile .
-    - docker push $IMAGE_NAME
+    - IMG=$DOCKER_REGISTRY/$PROJECTNAME:$TARGET_TAG make docker-build-ci docker-push
 
 push_release_tag:
   only:
     - tags
   stage: release
   tags: [ "runner:docker", "size:large" ]
   image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/docker-notary:0.6.1
+  before_script: []
   script:
     - IMAGE_NAME=$DOCKER_REGISTRY/$PROJECTNAME:$TARGET_TAG
     - RELEASE_IMAGE_NAME=$DOCKER_REGISTRY/$PROJECTNAME:$CI_COMMIT_TAG
     - docker pull $IMAGE_NAME
     - docker tag $IMAGE_NAME $RELEASE_IMAGE_NAME
     - docker push $RELEASE_IMAGE_NAME
 
-test_e2e:
```

Comment:
nit: let's rename the `unit_tests` job since it runs the new e2e tests from now on

Reply:
Done